react/features/app/actions.js

```diff
@@ -3,7 +3,7 @@
 import type { Dispatch } from 'redux';
 
 import { API_ID } from '../../../modules/API/constants';
-import { setRoom } from '../base/conference';
```

Comment:
unneeded change? if you are chaing the import, might as well import directly from actions.

Reply:
This was unintended . Will revert.